src/app/errors/components/ErrorCard.spec.tsx

```diff
@@ -1,14 +0,0 @@
-import React from 'react';
-import renderer from 'react-test-renderer';
-import MessageProvider from '../../MessageProvider';
-import ErrorCard from './ErrorCard';
-
-describe('ErrorCard', () => {
-
-  it('matches snapshot', () => {
-    const tree = renderer
-      .create(<MessageProvider><ErrorCard /></MessageProvider>)
```

Comment:
I'm not sure what do you mean. ErrorCard doesn't exist anymore, how can there be a spec for it? Should i split `Modal` into `Modal` and `Card`?

Reply:
no i was looking at the wrong thing. i should have been looking at `src/app/errors/components/__snapshots__/ErrorModal.spec.tsx.snap`. because you're refactoring the modal components that snapshot is useful because you've totally replaced the implementation of the component but it should still be the same in the snapshot. looking there now, i think everything looks good